pkg/feature/templates/serverless/serving-install/knative-serving.tmpl

```diff
@@ -0,0 +1,23 @@
+apiVersion: operator.knative.dev/v1beta1
+kind: KnativeServing
+metadata:
+  name: {{ .Serving.Name }}
+  namespace: knative-serving
+  annotations:
+    serverless.openshift.io/default-enable-http2: "true"
+spec:
+  deployments:
+    - annotations:
+        sidecar.istio.io/inject: "true"
+        sidecar.istio.io/rewriteAppHTTPProbers: "true"
+      name: activator
+    - annotations:
+        sidecar.istio.io/inject: "true"
+        sidecar.istio.io/rewriteAppHTTPProbers: "true"
+      name: autoscaler
+  ingress:
+    istio:
+      enabled: true
+  config:
+    istio:
+      local-gateway.knative-serving.knative-local-gateway: "knative-local-gateway.{{ .Mesh.Namespace }}.svc.cluster.local"
```

Comment:
If you rename struct in the API to `ControlPlane` remember to adjust the templates like this one :)